### ISSUE TITLE
[BUG] Elasticsearch synchronization is not always able to read in data from elastic

### ISSUE THREAD

**Issue by nagi49000** [memgraph/mage#503]
**Describe the bug**
On Elastic 8.14.3, reading docs from elastic with search() or scan() fails with a Python KeyError of "index" not found

**To Reproduce**

1. Connect to an 8.14.3 elastic instance from memgraph Cypher editor
2. In memgraph Cypher editor, run a query to get data from an index in that elastic index
```
CALL elastic_search_serialization.search("<some index in the elastic instance>",  "{\"match_all\": {}}", 1000, 0)
YIELD *
RETURN *;
```

**Expected behavior**
Return a valid response from the elasticsearch query without error

**Additional context**
Fix is straightforward. In the two functions `search()` and `scan()` in `elastic_search_serialization.py`, there is a code block reading (in search)
```
    for hit in response[HITS][HITS]:
        hit[ID] = hit[_SOURCE][INDEX][ID]
        hit[_SOURCE].pop(INDEX, None)
        hits.append(hit)
```
and (in scan)
```
    for item in response:
        item[ID] = item[_SOURCE][INDEX][ID]
        item[_SOURCE].pop(INDEX, None)
        items.append(item)
```

These blocks can be updated to (in search)
```
    for hit in response[HITS][HITS]:
        if hit[_SOURCE].get(INDEX, None):
            hit[ID] = hit[_SOURCE][INDEX][ID]
            hit[_SOURCE].pop(INDEX, None)
        hits.append(hit)
```
and (in scan)
```
    for item in response:
        if item[_SOURCE].get(INDEX, None):
            item[ID] = item[_SOURCE][INDEX][ID]
            item[_SOURCE].pop(INDEX, None)
        items.append(item)
```

**Comment by gitbuda** [comment]
Thanks a lot @nagi49000 for reporting!
Since the fix is simple, feel free to post a PR 🙏 
Hopefully, the team can pick this up quickly 💪

**Comment by nagi49000** [comment]
Thanks @gitbuda  - PR up https://github.com/memgraph/mage/pull/505

**Comment by gitbuda** [comment]
Fixed by #505
